Bhuvanesh/MMV_Make/components/Accordion.test.tsx

```diff
@@ -11,33 +9,77 @@ jest.mock("react", () => ({
 
 describe("Accordion Tests", () => {
   const setOpen = jest.fn((open: boolean) => !open);
-  const setHeight = jest.fn((height: number) => {});
+  const setView = jest.fn((index: number) => index);
   //   let open = jest.;
+  const data = {
```

Comment:
mock data can be saved in a different folder, so that the reusability can be increased